src/sentry/static/sentry/app/components/forms/selectControl.tsx

```diff
@@ -64,23 +96,27 @@ const SelectControl = props => {
 
   // Unfortunately we cannot use emotions `css` helper here, since react-select
   // *requires* object styles, which the css helper cannot produce.
-
-  const indicatorStyles = ({padding: _padding, ...provided}) => ({
+  const indicatorStyles = ({padding: _padding, ...provided}: React.CSSProperties) => ({
     ...provided,
     padding: '4px',
     alignItems: 'center',
     cursor: 'pointer',
   });
 
-  const defaultStyles = {
-    control: (_, state) => ({
+  const defaultStyles: StylesConfig = {
+    control: (_, state: any) => ({
       height: '100%',
       fontSize: '15px',
+      // @ts-ignore Ignore merging errors as only defining the property once
+      // makes code harder to understand.
       color: theme.formText,
       display: 'flex',
+      // @ts-ignore
       background: theme.background,
+      // @ts-ignore
       border: `1px solid ${theme.border}`,
       borderRadius: theme.borderRadius,
+      // @ts-ignore
       boxShadow: `inset ${theme.dropShadowLight}`,
```

Comment:
Can we put all these ignored props next to each other

maybe we could even do it as 

```tsx
// @ts-ignore
...{
	// Ignored props go in here
}
```